ui/panels/processors.php

```diff
@@ -315,7 +315,11 @@ function build_processor_types($default = null){
 	if(isset($config['template'])){
 		include $config['template'];
 	}else{
-		echo '<p>' . __('This processor has no configurable options.', 'caldera-forms') . '</p>';
+		if( isset($config['html'])){
```

Comment:
Instead of a nested `if`, should we instead use flatten the check into an `ifelse`?

Reply:
@kjohnson Yes, good point. I updated.